pull/context.go

```diff
@@ -255,4 +255,5 @@ type Body struct {
 	CreatedAt    time.Time
 	Author       string
 	LastEditedAt time.Time
+	ChangedFiles int
```

Comment:
Since it looks like nothing uses this field after calling the `Body()` method, I think should remove it for now and only add it in if it is required at some later point. The same thing would apply to the modification to the `v4PullRequestWithEditedAt` struct.